src/type/User.ts

```diff
@@ -0,0 +1,23 @@
+export interface User {
+  id: number;
+  name: string;
+  username: string;
+  email: string;
+  address: {
+    street: string;
+    suite: string;
+    city: string;
+    zipcode: string;
+    geo: {
+      lat: string;
+      lng: string;
+    };
+  };
+  phone: string;
+  website: string;
+  company: {
+    name: string;
+    catchPhrase: string;
+    bs: string;
+  };
+}
```

Comment:
```suggestion
export interface User {
  id: number;
  name: string;
  username: string;
  email: string;
}
```
actually, you don't need all those fields i this interface for this task